library-best-practices/JLBP-6.md

```diff
@@ -18,14 +18,15 @@ repository system, library B needs two identifiers to find classes in library A:
    which is formed from Maven artifacts and possibly non-Maven sources, is
    searched for each fully-qualified class name at runtime.
 
-When breaking changes are introduced to library A between major version 1 and
+When breaking changes are introduced to a library between major version 1 and
```

Comment:
"library A" was an intentional example name introduced up above.

Reply:
The principle applies to all libraries equally, B included, not to just library A